try {
    var m = 22;
    var ss = m/7 ;
        console.log(ss);
}
catch (e) {
    console.log("Error found" + e.message);
} 
finally {
     console.log("Value of a is 22 ");
 }
